src/Servers/IIS/build/assets.props

```diff
@@ -9,263 +9,261 @@
     </PropertyGroup>
     <ItemGroup>
        <Components
-          VCPlatform="x64"
-          Platform="x64"
-          ProjectName="AspNetCore.vcxproj"
-          NativeAsset="aspnetcore"
-          PropetyName="AspNetCoreModuleV1Shim"
-          BaseOutputPath="AspNetCoreModuleV1"
-          ProjectDirectory="$(MSBuildThisFileDirectory)..\AspNetCoreModuleV1\AspNetCore"
-          DllLocation="$(MSBuildThisFileDirectory)..\AspNetCoreModuleV1\AspNetCore\bin\$(Configuration)\x64\aspnetcore.dll"
-          Include="$(MSBuildThisFileDirectory)..\AspNetCoreModuleV1\AspNetCore\AspNetCore.vcxproj"
+          VCPlatform="x64" 
```

Comment:
Any reason why extra spaces were appended to each line?

Reply:
A side effect of how the script works. I wasn't able to find a good way to remove them.